packages/flutter_tools/lib/src/test/runner.dart

```diff
@@ -66,6 +66,7 @@ Future<int> runTests(
     shellPath: shellPath,
     watcher: watcher,
     enableObservatory: enableObservatory,
+    machine: json,
```

Comment:
Can you actually change the `json` argument to be named `machine` while you're in here?  It's weird that it starts out as `--machine` in `TestCommand`, then gets changed to `json`, then back to `machine`.